### PR TITLE
Create contact delete UI

### LINKED ISSUE #11: Create Contact Delete UI

### PR DESCRIPTION
**Pull request by chiragbagde**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({ onClose }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx:8]: `continue` with what? Not sure why I have this out again and again. Please go through the Slack thread.

**Review by yedhink — COMMENTED** [review]
@chiragbagde

File: app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx (hunk 1 above)
Comment (chiragbagde) [reply app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx:8]: Sorry I missed this part. I have corrected my mistake and updated the code.

**Comment by chiragbagde** [comment]
I've done the changes.

**Pull request merged**